project_script/main.py

```diff
@@ -82,19 +83,16 @@ def integrate_data():
 
 sanitize_data_frames()
 
-data_ramp_time = pd.concat(
-        [time_photo_celiv_ramp_time, odd_columns_subtracted],
-        axis=1
-    )\
+data_ramp_time = pd.concat([time_photo_celiv_ramp_time, odd_columns_subtracted], axis=1)\
```

Comment:
Suggest: if doesn't cross the limit length line, no need to break. Cause it's a simple constraint.
But, if cross or you need to put one more constraint, it's ok to break.